eng/common/templates/steps/send-to-helix.yml

```
@@ -0,0 +1,77 @@
+parameters:
+  HelixSource: 'pr/dotnet-github-anon-kaonashi-bot'     # required
+  HelixType: ̓'tests/default/'                           # required
```

There's a misplaced single quote here:

```suggestion
  HelixType: 'tests/default/'                           # required
```

---

fixed. that's wild.